### ISSUE TITLE
Pail can't handle non-comparable types

### ISSUE THREAD

**Issue by sritchie** [nathanmarz/dfs-datastores#4]
clojure.lang.PersistentHashMap, for example.

**Comment by nathanmarz** [comment]
What error do you get when you use a type like that?

**Comment by sritchie** [comment]
Here's a gist with my java and clojure code, and my example query & error: https://gist.github.com/1540755

This is the main exception:

``` java
;; Caused by: java.lang.ClassCastException: clojure.lang.PersistentArrayMap cannot be cast to java.lang.Comparable
;;  at cascading.tuple.Tuple.get(Tuple.java:221)
;;  at cascading.tuple.TupleEntry.get(TupleEntry.java:274)
;;  at backtype.cascading.tap.PailTap$PailScheme.sink(PailTap.java:106)
;;  at cascading.tap.Tap.sink(Tap.java:280)
;;  at cascading.flow.stack.SinkMapperStackElement.operateSink(SinkMapperStackElement.java:95)
;;  ... 27 more
```

**Comment by sritchie** [comment]
Fixed w/ dfs-datastores-cascading 1.1.1, released on Clojars.
